apps/wallet/src/ui/app/pages/home/tokens/coin-balance/index.tsx

```diff
@@ -37,17 +39,39 @@ function WalletBalanceUsd({ amount: walletBalance }: WalletBalanceUsdProps) {
 
 export function CoinBalance({ amount: walletBalance, type }: CoinProps) {
     const network = useAppSelector((state) => state.app.network);
-    const [formatted, symbol] = useFormatCoin(walletBalance, type);
+    const [formatted, symbol, { data: coinMetadata }] = useFormatCoin(walletBalance, type);
+
+    const iotaDecimals = coinMetadata?.decimals ?? 9;
+    const bnBalance = new BigNumber(walletBalance.toString()).shiftedBy(-1 * iotaDecimals);
+    const shouldShowTooltip = bnBalance.gt(0) && bnBalance.lt(1);
 
     return (
         <>
             <div className="flex items-baseline gap-0.5">
-                <div
-                    className="text-headline-lg text-neutral-10 dark:text-neutral-92"
-                    data-testid="coin-balance"
-                >
-                    {formatted}
-                </div>
+                {shouldShowTooltip ? (
+                    <Tooltip
+                        text={formatBalance(
+                            walletBalance,
+                            coinMetadata?.decimals ?? 9,
+                            CoinFormat.FULL,
+                        )}
+                        position={TooltipPosition.Bottom}
```

Comment:
Looks great! I’d just suggest positioning this to the right, but that’s just an idea, feel free to go with whatever feels best to you :saluting_face: 
```suggestion
                        position={TooltipPosition.Right}
```
![image](https://github.com/user-attachments/assets/4301e376-10f7-4249-a195-3fb53720cd10)

Reply:
I like it n the bottom, because on the right it covers the coin type